docs/contributing/index.md

```diff
@@ -1,69 +1,94 @@
 
-See our [support guidelines](../help/support-guidelines.md) for the detailed steps.
+To see the schedule for upcoming working group meetings, refer to the [Autoware Foundation events calendar](https://calendar.google.com/calendar/u/0/embed?src=autoware.org_6lol0ho5ft0217h8c60pi1fm30@group.calendar.google.com).
+
+### Reporting bugs
+
+Before you submit an issue, please search the issue tracker for the appropriate repository. It is possible that someone has already reported the same issue and that workarounds exist. If you can't determine the appropriate repository, ask the maintainers for help by creating a new discussion in the [Q&A category](https://github.com/autowarefoundation/autoware/discussions/categories/q-a).
+
+When reporting a bug, you should provide a minimal set of instructions to reproduce the issue. Doing so allows us to quickly confirm and focus on the right problem.
+
+If you want to fix the bug by yourself, then that will be appreciated, but you should discuss possible approaches with the maintainers first before submitting a pull request.
+
+You can file new issues by selecting from our new issue templates and filling out the issue template.
 
 ### Pull requests
 
-You can submit a pull request directly (without opening an issue) if it is small like the following:
+Pull requests can be submitted directly for small changes such as:
+
+- Minor documentation updates.
+- Fixing spelling mistakes.
+- Fixing CI failures.
+- Fixing warnings detected by compilers or analysis tools.
+- Making small changes to a single package.
 
-- Update a small part of documents.
-- Fix spelling mistakes.
-- Fix CI failures.
-- Fix warnings detected by compilers or analysis tools.
-- Make small changes to a single package.
+If your pull request is not covered by the examples above or is a large change, create an issue or a discussion thread to discuss possible approaches with the maintainers before submitting a pull request. Doing so ensures that the proposed change will be in line with Autoware's design philosophy and current development plans. Lastly, ensure that you use the appropriate pull request template for the target repository.
 
-Otherwise, create an issue or a discussion thread before submitting a pull request and discuss the approach with maintainers.
+We have some additional guidelines specific to the type of change being made, so please make sure to review those before submitting your pull request.
 
-Also, regardless of the size of your pull request, follow the pull request template of the target repository.
+- [Coding guidelines](coding-guidelines/index.md)
+- [Testing guidelines](testing-guidelines/index.md)
+- [Documentation guidelines](documentation-guidelines/index.md)
```

Comment:
These items duplicate the number 3 [here](https://autowarefoundation.github.io/autoware-documentation/pr-91/contributing/pull-request-guidelines/#general-pull-request-workflow).

Do you want to write them here as well?

Reply:
Ah. Good point. In that case I will remove these and just leave the link to the PR guidelines.